### ISSUE TITLE
代码报错

### ISSUE THREAD

**Issue by ZXU017** [TianLin0509/DNN_detection_via_keras#19]
name 'BatchNormalization' is not defined
这个问题是什么原因呢

**Comment by nono741852** [comment]
tensorflow版本的关系，试试在main.py函数的导入库改成
from tensorflow.keras.layers import Input, Dense, BatchNormalization
from tensorflow.keras import Model, callbacks
from generations import *
import tensorflow as tf
